lib/rules/jsx-first-prop-new-line.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
               report(context, messages.propOnNewLine, 'propOnNewLine', {
                 node: decl,
                 fix(fixer) {
-                  return fixer.replaceTextRange([(node.typeParameters || node.name).range[1], decl.range[0]], '\n');
+                  return fixer.replaceTextRange([(node.typeArguments || node.typeParameters || node.name).range[1], decl.range[0]], '\n');
```

Comment:
having `node.typeParameters` in the middle here will trigger the deprecation warning if it should ever fall back to `node.name`. is that anything to be concerned about?

Reply:
good point - so in v6, `typeArguments` will be conditionally present, but `typeParameters` unconditionally a warning getter?